resources/lib/dplay.py

```diff
@@ -833,7 +833,13 @@ def get_stream(self, video_id, video_type):
 
         # discoveryplus.com (US)
         if self.locale_suffix == 'us':
-            stream['hls_url'] = data_dict['attributes']['streaming'][0]['url']
+            originalm3u8 = requests.get(data_dict['attributes']['streaming'][0]['url']).text
+            updatedm3u8 = originalm3u8.replace("30.000", "29.970")
```

Comment:
Can you add comment here why this is done? :) Something like discovery+ says wrong frame...

Reply:
done